bin/deploy_wrapper.rb

```diff
@@ -31,7 +31,9 @@
 
   command = "./bin/deploy.sh -f #{image_path} -n \"#{openstack_image_name}\" -r #{options[:pr_number]}"
   # when the chef_version is not specified in the template, don't use it!
-  command += "-c #{chef_version}" unless chef_version.nil?
+  command += " -c #{chef_version}" unless chef_version.nil?
+  # publish image if option is set
+  command += ' -p' if options[:publish]
```

Comment:
Would `-p` in combination with other values in the command still work?

Reply:
I would think so, optionparser is pretty good. I will test it.